admin/includes/functions/general.php

```diff
@@ -2666,6 +2666,17 @@ function zen_get_configuration_key_value($lookup) {
     return $lookup_value;
   }
 
+  function zen_get_configuration_group_value($lookup) {
+    global $db;
+    $configuration_query= $db->Execute("select configuration_group_title from " . TABLE_CONFIGURATION_GROUP . " where configuration_group_id ='" . zen_db_input($lookup) . "'");
+    $lookup_value= $configuration_query->fields['configuration_group_title'];
+    if ( $configuration_query->RecordCount() == 0 ) {
+      $lookup_value='<span class="lookupAttention">' . $lookup . '</span>';
```

Comment:
Why the insertion of HTML here, when the only use of this function (presently) is to return a string that gets output as part of a page `<title>` and thus wouldn't want nested `<span>`s in it?